src/services/suite.service.js

```diff
@@ -1,9 +1,15 @@
 import Alert from '@/model/Alert.model'
 import { createApolloClient } from '@/utils/graphql'
-import axios from 'axios';
 import gql from 'graphql-tag'
 import store from '@/store/'
-import Suite from '@/model/Suite.model';
+
+// query to retrieve all suites
+const suitesQuery = gql`query {
+  workflows {
+    id, name, owner, host, port
+  }
+}
```

Comment:
The GraphQL query. `gql` is a special syntax which some day I will distill to understand how that works in JavaScript. But that comes from this [apollo library to parse graphql queries](https://github.com/apollographql/graphql-tag).